.gitignore

```diff
@@ -0,0 +1 @@
+.DS_store
```

Comment:
please review files before committing and don't modify anything outside of your student directory.